solidity/random-beacon/test/Utils.test.ts

```diff
@@ -0,0 +1,75 @@
+import { ethers } from "hardhat"
```

Comment:
Could we move the file to `solidity/random-beacon/test/utils/dkg.test.ts`.
It feels to me that placing it next to all the contract tests may cause a confusion.

Reply:
2ca346e